apps/api/src/app/environments-v1/usecases/output-renderers/email-output-renderer.usecase.ts

```diff
@@ -47,11 +57,17 @@ export class EmailOutputRendererUsecase {
       };
     }
 
-    const liquifiedMaily = wrapMailyInLiquid(body);
-    const transformedMaily = await this.transformMailyContent(liquifiedMaily, renderCommand.fullPayloadForRender);
-    const parsedMaily = await this.parseMailyContentByLiquid(transformedMaily, renderCommand.fullPayloadForRender);
-    const strippedMaily = this.removeTrailingEmptyLines(parsedMaily);
-    const renderedHtml = await mailyRender(strippedMaily);
+    let renderedHtml: string;
+
+    if (typeof body === 'object' || (typeof body === 'string' && isJsonString(body))) {
+      const liquifiedMaily = wrapMailyInLiquid(body);
+      const transformedMaily = await this.transformMailyContent(liquifiedMaily, renderCommand.fullPayloadForRender);
+      const parsedMaily = await this.parseMailyContentByLiquid(transformedMaily, renderCommand.fullPayloadForRender);
+      const strippedMaily = this.removeTrailingEmptyLines(parsedMaily);
+      renderedHtml = await mailyRender(strippedMaily);
+    } else {
+      renderedHtml = body;
```

Comment:
we dont need to compile by liquid because if will be rendered as output in the framework.